src/main/java/org/paymentsds/mpesa/internal/MpesaRequest.java

```diff
@@ -0,0 +1,157 @@
+package org.paymentsds.mpesa.internal;
+
+import org.paymentsds.mpesa.Request;
+
+public class MpesaRequest {
+
+    private String input_TransactionReference;
+    private String input_Amount;
+    private String input_ThirdPartyReference;
+    private String input_CustomerMSISDN;
+    private String input_ServiceProviderCode;
+
+    // B2B
+    private String input_PrimaryPartyCode;
+    private String input_ReceiverPartyCode;
+
+    // Reversal
+    private String input_ReversalAmount;
+    private String input_InitiatorIdentifier;
+    private String input_SecurityCredential;
+    private String input_TransactionID;
```

Comment:
Do we need to use snake case here?

Reply:
@edsonmichaque not really. We can use camelCase and then add the `SerializedName` annotation to each field, e.g:

```java
@SerializedName("input_ServiceProviderCode")
private String inputServiceProviderCode;
```

What do you think?

Reply:
@rosariopfernandes I like the idea

Reply:
Done in b6823d3